src/System/Boot.tsx

```diff
@@ -63,6 +65,18 @@ export const Boot: React.FC<
     ...context,
   }
 
+  const unleashConfig = {
+    url: `${sd.UNLEASH_API}frontend/`,
+    clientKey: sd.UNLEASH_CLIENT_KEY,
+    refreshInterval: 15,
+    appName: sd.UNLEASH_APP_NAME,
+    environment: sd.NODE_ENV,
```

Comment:
I think the only environments we support on the Unleash side are `development` and `production`. Should we coerce this value into one of those supported values?

We could infer from NODE_ENV but it be clearer and more flexible to use an explicit `UNLEASH_ENVIRONMENT` config value which we can default to `development` unless targeting a production environment.

Reply:
Agree! good call out. Was conflating runtime environment with how we've configured `unleash`'s environment. Changed [229a483](https://github.com/artsy/force/pull/15263/commits/229a483081f47867c472e9492fabf9e04dfc0562)